src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -696,6 +696,8 @@ protected static void setFormat(StreamType streamType, DLNAMediaInfo media, DLNA
 				format = FormatConfiguration.VC1;
 		} else if (value.equals("au") || value.equals("uLaw/AU Audio File")) {
 			format = FormatConfiguration.AU;
+		} else if (value.equals("av01")) {
```

Comment:
The string I can see in the MediaInfo output is `av1` not `av01`, is there a reason to have it like this?

Reply:
@SubJunk `AV1` is a Format and Codec ID is `av01`. I updated the parsing